packages/terra-card/src/Card.scss

```diff
@@ -0,0 +1,27 @@
+
+/* stylelint-disable selector-class-pattern  */
+.card {
+  background-color: #fff;
+  border-radius: 3px;
+  box-shadow: 0 1px 1px 0 #1c1f21;
+  margin: 10px;
```

Comment:
I would say we should remove the margin for now and a themeable option could be added later if a need for it occurs.